src/System.Globalization/tests/CultureInfo/CultureInfoClone.cs

```diff
@@ -2,39 +2,22 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using System;
-using System.Globalization;
 using Xunit;
 
 namespace System.Globalization.Tests
 {
     public class CultureInfoClone
     {
-        [Fact]
-        public void TestClone1()
+        [Theory]
+        [InlineData("")]
+        [InlineData("fr-FR")]
+        [InlineData("en")]
+        public void TestClone(string name)
         {
-            CultureInfo myCultureInfo = new CultureInfo("fr-FR");
-            CultureInfo myClone = myCultureInfo.Clone() as CultureInfo;
-            Assert.True(myClone.Equals(myCultureInfo));
-            Assert.NotSame(myClone, myCultureInfo);
-        }
-
-        [Fact]
-        public void TestClone2()
-        {
-            CultureInfo myCultureInfo = new CultureInfo("en");
-            CultureInfo myClone = myCultureInfo.Clone() as CultureInfo;
-            Assert.True(myClone.Equals(myCultureInfo));
-            Assert.NotSame(myClone, myCultureInfo);
-        }
-
-        [Fact]
-        public void TestClone3()
-        {
-            CultureInfo myTestCulture = CultureInfo.InvariantCulture;
-            CultureInfo myClone = myTestCulture.Clone() as CultureInfo;
-            Assert.True(myClone.Equals(myTestCulture));
-            Assert.NotSame(myClone, myTestCulture);
+            CultureInfo culture = new CultureInfo(name);
```

Comment:
I realize that "" represents an invariant culture, but it'd be good not to lose the test for CultureInfo.InvariantCulture explicitly.